exercises/dominoes/dominoes.rb

```diff
@@ -0,0 +1,8 @@
+class Dominoes
+
+  def can_chain?(dominoes_list = [])
+    result = false
+    return true if dominoes_list.length == 0 
+    result
```

Comment:
Ah this is true, but I do not believe and of the solutions beyond the base cases are dependent on length. I just wanted to get the base cases out of the way.

My suspicion is that I need to frame finding the generic solution as a graph, and then figure out how to traverse the graph to see if at least one solution exists..



"Serena Sees Her Footprints on the Moon" is on sale now @ iTunes! 
https://itun.es/us/CnX6bb.l

It's the story of a girl who imagines a trip to the Moon, where she interacts with some of the objects left behind up there by the astronauts from the Apollo missions.

No iPad or iPhone? No problem!
Get the book here: https://gum.co/softcover

> On Aug 28, 2016, at 6:10 PM, Tute Costa <notifications@github.com> wrote:
> 
> In exercises/dominoes/dominoes.rb:
> 
> > @@ -0,0 +1,8 @@
> > +class Dominoes
> > +
> > +  def can_chain?(dominoes_list = [])
> > +    result = false
> > +    return true if dominoes_list.length == 0 
> > +    result
> Note: Ruby defines the zero? method on numbers:
> 
> % irb
> irb(main):001:0> 1.zero?
> => false
> irb(main):002:0> 0.zero?
> => true
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>